Source/Core/Common/BitUtils.h

```diff
@@ -99,4 +99,26 @@ constexpr Result ExtractBits(const T src) noexcept
 
   return ExtractBits<T, Result>(src, begin, end);
 }
+
+///
+/// Verifies whether the supplied value is a valid bit mask of the form 0b00...0011...11.
+/// Both edge cases of all zeros and all ones are considered valid masks, too.
+///
+/// @param  mask The mask value to test for validity.
```

Comment:
Yeah, the original code came from a utility file from an [ARMv7](https://bitbucket.org/Lioncash/forearm/src/963a94b5011e2f5a2d0e874d63118430c8a3e888/libForeARM/src/Utils/Bits.hpp?at=master&fileviewer=file-view-default) interpreter that I'm working on.